source/elements/variables/colors.css

```diff
@@ -4,31 +4,35 @@
 
   background-color: var(--color-black);
   color: var(--color-white);
+
+  IMPORTANT:
+  In order for the project colors to be included by the SgColorSwatch,
+  please prefix all variables inside this file with '---color'.
 */
 
 :root {
```

Comment:
Until you restart dev, any renaming of colors in here is not replacing colors in SgColorSwatch_variables.json, but instead only adding colors to it. This leaves you in a state where you have colors in SgColorSwatch__variables.json that don't actually exsist in project anymore.

Reply:
fixed by recent commits